kong/core/balancer.lua

```diff
@@ -309,6 +310,7 @@ local function execute(target)
 
   target.ip = ip
   target.port = port
+  target.hostname = target.host
```

Comment:
Why not using `host` or rename `host` to `hostname`? They seem to ultimately be identical even though the comments in the `balancer_address` structure seems to say otherwise.